src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -171,6 +171,9 @@ public String url() {
 
     private transient boolean allowKill;
 
+    /** Controls whether or not our execution has been initialized via its {@link FlowExecution#onLoad(FlowExecutionOwner)} method yet if.*/
+    transient boolean executionLoaded = false;  // NonPrivate for tests
```

Comment:
I actually wonder if that might be something we want to expose outside of just tests - I don't have a specific scenario in mind, but I know I've stumbled across cases where this could be useful information. I think. I may be getting confused with something else, though.

Reply:
Can always expose it with a getter.